foundry.toml

```
@@ -0,0 +1,12 @@
+[profile.default]
+src = 'src'
+out = 'out'
+libs = ['lib']
+
+gas_reports = ["TodoList"]
```

```suggestion
gas_reports = ["TodoList"]
solc_version = '0.8.23'
evm_version = 'shanghai'
```

`evm_version` to match hedera's hyperledger besu version
`solc_version` to to maximise longevity of/ minimise maintenance requirements on tutorial - see detailed comment: https://github.com/hedera-dev/preview-hedera-docs/pull/15#discussion_r1480797343